mgr/src/server/plugins/backup_and_restore.cr

```diff
@@ -0,0 +1,49 @@
+require "moana_types"
+require "file_utils"
+require "../conf"
+require "./helpers"
+require "../datastore/*"
+require "./ping"
+require "./volume_utils.cr"
+
+post "/api/v1/backup" do |env|
```

Comment:
Change this URL to `/api/v1/backups`. Also change URL for restore as `/api/v1/backups/:backup_name/restore`. With this more API functionalities can be added later. For example,

```
DELETE /api/v1/backups/:name
```